sbt-plugin/src/main/scala/org/scalajs/sbtplugin/Implicits.scala

```diff
@@ -5,7 +5,9 @@ import scala.language.implicitConversions
 import org.scalajs.core.tools.logging._
 import sbt.{Logger => SbtLogger, Level => SbtLevel}
 
+@deprecated("Use sbtplugin.Loggers explicitly instead.", "0.6.10")
 object Implicits {
+  @deprecated("Use sbtplugin.Loggers.SbtLoggerWrapper instead.", "0.6.10")
   private class SbtLoggerWrapper(underlying: SbtLogger) extends Logger {
```

Comment:
Why do you keep `SbtLoggerWrapper` at all? It's private, so we can freely remove it.